### PR TITLE
hyperlink in log

### PR DESCRIPTION
**Pull request by torusrxxx**
Addresses will become clickable hyperlinks

---
This change is [Reviewable](https://reviewable.io/reviews/x64dbg/x64dbg/1070)

### CODE REVIEW

**Comment by torusrxxx** [comment]
#1060

**Comment by mrexodia** [comment]
very nice <3

File: src/gui/Src/Gui/LogView.cpp [hunk 1]
```diff
@@ -112,9 +140,54 @@ void LogView::addMsgToLogSlot(QString msg)
...
+            duint address = link.fragment(QUrl::DecodeReserved).toULongLong(&ok, 16);
+            if(ok && DbgMemIsValidReadPtr(address))
+            {
+                if(DbgFunctions()->MemIsCodePage(address, true))
+                    DbgCmdExec(QString("disasm %1").arg(link.fragment()).toUtf8().constData());
+                else
+                    DbgCmdExec(QString("dump %1").arg(link.fragment()).toUtf8().constData());
+            }
+            else
+                SimpleErrorBox(this, "1", QString("disasm %1").arg(link.fragment()) + link.toDisplayString());
```
Comment (torusrxxx) [inline src/gui/Src/Gui/LogView.cpp]: remove this debug output

File: src/gui/Src/Gui/LogView.cpp [hunk 2]
```diff
@@ -112,9 +140,54 @@ void LogView::addMsgToLogSlot(QString msg)
...
+        }
+        else if(link.path() == "/address32")
+        {
+            bool ok = false;
+            duint address = link.fragment(QUrl::DecodeReserved).toULong(&ok);
+            if(ok)
```
Comment (torusrxxx) [inline src/gui/Src/Gui/LogView.cpp]: Please keep 32-bit code and 64-bit code the same

**Pull request merged**